classes/shortcodes.php

```diff
@@ -197,13 +228,18 @@ function do_shortcode( $args ) {
 			}
 
 			// If the requested data is a taxonomy
-			$sm_taxes = $simple_map->get_sm_taxonomies( 'array', '', true, 'objects' ) ? $simple_map->get_sm_taxonomies( 'array', '', true, 'objects' ) : array();
+			$sm_taxes = $simple_map->get_sm_taxonomies( 'array', '', true,
+				'objects' ) ? $simple_map->get_sm_taxonomies( 'array', '', true,
+				'objects' ) : array();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Closing parenthesis of a multi-line function call must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->